dataprofiler/reports/graphs.py

```diff
@@ -59,7 +70,7 @@ def plot_histograms(profiler, column_names=None, column_inds=None):
     if not column_names and not column_inds:
         inds_to_graph = list(range(len(profiler.profile)))
     elif not column_inds:
-        for column in column_names:
+        for column in cast(list, column_names):
```

Comment:
Technically this is getting cast as a `List[Any]`. Casting to `List[Union[str, list]]` doesn't work because `_col_name_to_idx` in `StructuredProfiler` is cast to `List[any]` as well. So if we cast to `List[Union[str, int]]`, we can get an error saying the `List[Any]` and `List[Union[str, int]]` are not compatible.

Reply:
This should match better but at the same time want to keep the scope smaller for this PR.

Reply:
Does fixing the one line in the other file fix it though?

Reply:
unfortunately, it didn't work when I tested that locally